ts/ts-bas/src/test/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionIdentitetStyrktGenomTest.java

```diff
@@ -0,0 +1,225 @@
+                        .build(),
+                    RadioMultipleCode.builder()
+                        .id(IdKontrollKod.FORETAG_ELLER_TJANSTEKORT.getCode())
+                        .label(textProvider.get(IDENTITET_STYRKT_GENOM_FORETAG_ELLER_TJANSTEKORT_TEXT_ID))
+                        .build(),
+                    RadioMultipleCode.builder()
+                        .id(IdKontrollKod.KORKORT.getCode())
+                        .label(textProvider.get(IDENTITET_STYRKT_GENOM_KORKORT_TEXT_ID))
+                        .build(),
+                    RadioMultipleCode.builder()
+                        .id(IdKontrollKod.PERS_KANNEDOM.getCode())
+                        .label(textProvider.get(IDENTITET_STYRKT_GENOM_PERS_KANNEDOM_TEXT_ID))
+                        .build(),
+                    RadioMultipleCode.builder()
+                        .id(IdKontrollKod.FORSAKRAN_KAP18.getCode())
+                        .label(textProvider.get(IDENTITET_STYRKT_GENOM_FORSAKRAN_KAP18_TEXT_ID))
+                        .build(),
+                    RadioMultipleCode.builder()
+                        .id(IdKontrollKod.PASS.getCode())
+                        .label(textProvider.get(IDENTITET_STYRKT_GENOM_PASS_TEXT_ID))
+                        .build()
+                );
+            }
+
+            @Override
+            protected Layout getExpectedLayout() {
+                return COLUMNS;
+            }
+        }
+
+        @Nested
+        class IncludeValueCodeTest extends ValueCodeTest {
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionIdentitetStyrktGenom.toCertificate(
+                    Vardkontakt.create(null, IdKontrollKod.FORETAG_ELLER_TJANSTEKORT.getCode()), 0, textProvider);
+            }
+
+            @Override
+            protected CertificateDataValueCode getCertificateDataValueCode() {
+                return CertificateDataValueCode.builder()
+                    .id(IdKontrollKod.FORETAG_ELLER_TJANSTEKORT.getCode())
+                    .code(IdKontrollKod.FORETAG_ELLER_TJANSTEKORT.getCode())
+                    .build();
+            }
+        }
+
+        @Nested
+        class IncludeValidationMandatoryTest extends ValidationMandatoryTest {
+
+            @Override
+            protected String getQuestionId() {
+                return IDENTITET_STYRKT_GENOM_SVAR_ID_2;
+            }
+
+            @Override
+            protected String getExpression() {
+                return "$IDK1 || $IDK2 || $IDK3 || $IDK4 || $IDK5 || $IDK6";
+            }
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionIdentitetStyrktGenom.toCertificate(null, 0, textProvider);
+            }
+
+            @Override
+            protected int getValidationIndex() {
+                return 0;
+            }
+        }
+
+        @Nested
+        class ToInternal {
```

Comment:
There is a testFixure called ValuePair that could be used for ToInternal tests like this one. Im currently working on it in my branch. Hit me up and we can discuss how to apply it to this class.